### DISCUSSION TITLE
image labelling

### DISCUSSION THREAD

**Discussion by vasnt** [cleanlab/cleanlab#733] [Q&A]
Cleanlab can be used to assign labels for images?

**Comment by jwmueller** [comment]
Cleanlab can help you decide what images would be most informative for you (or your labeling team) to label, i.e. what is called active learning:  https://cleanlab.ai/blog/active-learning/

Cleanlab is also very useful in help you decide which labels are worth a second a look because they appear likely incorrect (reviewing dataset labels). 

If you're just interested in predicting the labels of images in test data, that would simply be a standard machine learning task. Cleanlab can help you train better models for this ML task (by improving the training dataset), but it is not a ML framework to produce such predictions directly. See more about this here: https://github.com/cleanlab/cleanlab/discussions/696
